pkgs/by-name/fu/fum/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  autoPatchelfHook,
+  stdenv,
+  openssl,
+  dbus,
+  pkg-config,
+  libgcc,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "fum";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "qxb3";
+    repo = "fum";
+    rev = "v${version}";
+    hash = "sha256-VRcQWwO80xFn5A21yjRsGqnnWkhWfsJxxEiw78NWJPM=";
+  };
+
+  cargoHash = "sha256-GW3/SqQlEUTMtvOgnMGhcREOHz/V2qtjtCAzFFKMNb4=";
+
+  nativeBuildInputs = lib.optionals stdenv.isLinux [
+    autoPatchelfHook
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+    dbus
+    libgcc
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A fully ricable tui-based music client";
```

No need for indefinite articles in the description :)
```suggestion
    description = "Fully ricable tui-based music client";
```